test/functional/Panama/README.md

```diff
@@ -18,20 +18,4 @@ OpenJDK Assembly Exception [2].
 [2] https://openjdk.org/legal/assembly-exception.html
 
 SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
--->
-
-# Panama Test
-- Contains [tests](src/org/openj9/test/panama/) for native method handles
-- Build with the "Build git tests - panama" job
-- Run with linux_x86-64_cmprssptrs_panama.spec on the rawbuild-panama_xa64-b136 sdk
-- [HigherLevelNativeMethodHandleTest](src/org/openj9/test/panama/HigherLevelNativeMethodHandleTest.java) uses [panamatest.jar](lib/panamatest.jar), created from [panamatest.c](panamatest.c) with the jextract groveller
-
-
-    * To create a libpanamatest.so, a gcc is needed:
-
-            gcc -shared -o {$common.workspace.gittest$}/Panama/libpanamatest.so -fPIC {$common.workspace.gittest$}/Panama/panamatest.c
-
-    * To create a file like panamatest.jar, on a machine with os:linux:ubuntu14 capability (such as ub1404x64vm1), run:
-
-            export LD_LIBRARY_PATH=<path_to_clang_lib>/clang/clang+llvm-3.8.0-x86_64-linux-gnu-ubuntu-14.04/lib
-            <OpenJDK_root>/bin/jextract -J-Djava.library.path=<path_to_clang_lib>/clang -t panama.test -o panamatest.jar <path to the test repo>/Panama/panamatest.c
+-->
```

Comment:
I expect we'll want to simply delete the `test/functional/Panama` folder. @ChengJin01 Do you agree?

Reply:
The test cases in the folder were initially intended for Native MethodHandle in FFI which we'd like to take reference to when we are implementing the JIT path in downcall. So I'd suggest to keep it temporarily once everything with VM/JIT is implemented (in which case the test cases might be absorbed to our own FFI test suites at e.g. `test/functional/Java21andUp/src/org/openj9/test/jep442`).

In addition, it'd better to keep everything related to Native MethodHandle in this PR for the time being in case anything we might ignore in the future.

Reply:
Some of these tests are already redundant, and the code in the `Panama` folder will always be accessible in `git` history, I think now is the time to remove this (the whole `test/functional/Panama` folder).